tests/test.hs

```diff
@@ -1,48 +1,61 @@
 {-# LANGUAGE OverloadedStrings #-}
 
-import Data.List.NonEmpty (fromList)
-import Data.Text as T
-import Network.SendGridV3.Api
-import System.Environment
-import Test.Tasty
-import Test.Tasty.HUnit
-import Control.Lens ((^.))
-import Network.Wreq
+import           Control.Lens                             ( (^.) )
+import           Data.List.NonEmpty                       ( fromList )
+import           Data.Text                     as T
+import           Network.SendGridV3.Api
+import           Network.Wreq
+import           System.Environment
+import           Test.Tasty
+import           Test.Tasty.HUnit
 
 testMail :: MailAddress -> Mail () ()
-testMail addr =
-  mail [personalization (fromList [addr])] addr "Mail Subject" (fromList [mailContentText "Test Content"])
+testMail addr = mail [personalization (fromList [addr])]
+                     addr
+                     "Mail Subject"
+                     (fromList [mailContentText "Test Content"])
 
 main :: IO ()
 main = do
   sendgridKey  <- getSendGridKey
   testMailAddr <- getTestEmailAddress
-  defaultMain $ testGroup "SendGrid v3 API"
-    [
-      testCase "Send email simple" $ do
-        eResponse <- sendMail sendgridKey (testMail testMailAddr)
-        case eResponse of
-          Left err -> error "Failed to send simple email"
-          Right r -> r ^. responseStatus . statusCode @?= 202
+  defaultMain $ testGroup
```

Comment:
this went through an autoformatter. Let  me know if you want me to revert the style changes; it just automatically happened on-save and I figured it'd be easier to push and ask than manually revert and rewrite without autoformatting. :grimacing:

Reply:
That's fine definitely looks better than before especially the imports.